marlowe-playground-client/static/css/main.scss

```diff
@@ -201,4 +203,23 @@ button.start-simulation-btn {
 
 .monaco-readonly-decoration {
     background-color: #fafafa;
-}
\ No newline at end of file
+}
+
+// TODO: See if we can enable this clases only for
+// development, remove otherwise
```

Comment:
if you put them in a separate file I'm sure you could get webpack to find that file only in dev and find an empty file otherwise (using aliases perhaps)

Reply:
I never configured webpack before, for the moment I prefer to remove it (it's easy to find it online)